### ISSUE TITLE
Zoomed out pins

### ISSUE THREAD

**Issue by ghost** [jacobwalkr/caerus-web#4]
http://google-maps-utility-library-v3.googlecode.com/svn/tags/markerclusterer/

Have you seen this?

**Comment by ghost** [comment]
A tool for clustering markers on a map.

**Comment by ghost** [comment]
Refer to this documentation:
http://google-maps-utility-library-v3.googlecode.com/svn/trunk/markerclusterer/docs/examples.html

**Comment by jacobwalkr** [comment]
He was just offering a suggestion.

**Comment by jacobwalkr** [comment]
Marker Clusterer is what the YRSNYC winners used to display clusters on their map - it makes it really easy.

**Comment by jacobwalkr** [comment]
What's good about it is ease of use - somebody's done all the hard work for you. It's good to practice, but sometimes it's not worth reinventing the wheel.

**Comment by jacobwalkr** [comment]
Yeah, alright, fair enough. He was just making a suggestion. I thought it was helpful - it was something we discussed at the Festival.

Out of interest, why does your approach suit our purposes better?
